nilearn/datasets/struct.py

```diff
@@ -106,38 +120,126 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
+       DOI: 10.1016/j.neuroimage.2010.07.033
+
+    .. [2] VS Fonov, AC Evans, RC McKinstry, CR Almli and DL Collins, Unbiased
+       nonlinear average age-appropriate brain templates from birth to
+       adulthood, NeuroImage, Volume 47, Supplement 1, July 2009, Page S102
+       Organization for Human Brain Mapping 2009 Annual Meeting,
+       DOI: 10.1016/S1053-8119(09)70884-5
 
     """
-    return check_niimg(MNI152_FILE_PATH)
 
+    brain_template = check_niimg(MNI152_FILE_PATH)
+
+    # Change the resolution of the template
+    if resolution != 1:
+        brain_template = resampling.resample_img(brain_template,
+                                                 np.eye(3) * resolution)
+
+    return brain_template
+
+
+def load_mni152_gm_template(resolution=2):
+    """This function takes the skullstripped 1mm-resolution version of the
+    gray-matter MNI152 template, originally distributed with FSL, and
+    re-samples it using a different resolution, if specified.
+
+    Parameters
+    ----------
+    resolution: int, optional, Default = 2
+        If resolution is different from 1, the template is re-sampled with the
+        specified resolution.
+
+    Returns
+    -------
+    gm_mni152_template : Nifti1Image, image representing the resampled
+        gray-matter template
+
+    """
 
-def load_mni152_brain_mask():
-    """Load brain mask from MNI152 T1 template
+    gm_template = check_niimg(GM_MNI152_FILE_PATH)
+
+    # Change the resolution of the template
+    if resolution != 1:
+        gm_template = resampling.resample_img(gm_template,
+                                              np.eye(3) * resolution)
+
+    return gm_template
+
+
+def load_mni152_wm_template(resolution=2):
+    """This function takes the skullstripped 1mm-resolution version of the
+    white-matter MNI152 template, originally distributed with FSL, and
+    re-samples it using a different resolution, if specified.
+
+    Parameters
+    ----------
+    resolution: int, optional, Default = 2
+        If resolution is different from 1, the template is re-sampled with the
+        specified resolution.
+
+    Returns
+    -------
+    wm_mni152_template : Nifti1Image, image representing the resampled
+        white-matter template
+
+    """
+
+    wm_template = check_niimg(WM_MNI152_FILE_PATH)
+
+    # Change the resolution of the template
+    if resolution != 1:
+        wm_template = resampling.resample_img(wm_template,
+                                              np.eye(3) * resolution)
+
+    return wm_template
+
+
+def load_mni152_brain_mask(resolution=2, threshold=0.2):
+    """Load mask from the whole-brain MNI152 T1 template.
+
+    Parameters
+    ----------
+    resolution: int, optional, Default = 2
+        If resolution is different from 1, the template loaded is first
+        re-sampled with the specified resolution.
 
-    .. versionadded:: 0.2.5
+    threshold : float, optional
```

Comment:
`resolution` and `threshold` are both new parameters of this function. I would also add the versionadded sphinx directive for both.